types/subleveldown/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for subleveldown 4.1
+// Project: https://github.com/level/subleveldown
+// Definitions by: Carson Farmer <https://github.com/carsonfarmer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { LevelUp } from 'levelup';
+import { ErrorValueCallback, ErrorCallback, AbstractGetOptions } from 'abstract-leveldown';
+
+declare namespace sub {
+    interface Options {
+        separator?: string;
+        open?: ((callback?: ErrorCallback) => any);
```

Comment:
[Why is callback listed as optional?][optional-parameters-in-callbacks] In [source code][_open], it is always passed.

[optional-parameters-in-callbacks]: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#optional-parameters-in-callbacks
[_open]: https://github.com/Level/abstract-leveldown/blob/52d0179e272f2f5c0420f3e24415510aa63e042b/abstract-leveldown.js#L34

Reply:
Good point, and thanks for the link! Fixed.